From f80b5ba64d3a40ecd7a35b6c5c3e5664b74af769 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 30 May 2020 20:01:42 +0300
Subject: [PATCH 12/12] Revert "arm a is even tighter, so give only 8MB
 headroom"

This reverts commit bcb48d0ba8c6473e31c31bb7fbdfd5b7efc30da8.

  need this to fix build LOS

  ERROR: Failed to build
  out/target/product/phhgsi_arm_a/obj/PACKAGING/systemimage_intermediates/system.img
  from out/target/product/phhgsi_arm_a/system
  Out of space? Out of inodes?
  The tree size of out/target/product/phhgsi_arm_a/system is 1294205952 bytes (1234 MB),
  with reserved space of 8388608 bytes (8 MB).
  The max image size for filesystem files is 1302597632 bytes (1242 MB),
  out of a total partition size of 1302597632 bytes (1242 MB).
---
 phhgsi_arm_a/BoardConfig.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/phhgsi_arm_a/BoardConfig.mk b/phhgsi_arm_a/BoardConfig.mk
index 9d8e5d5..36a8bf4 100644
--- a/phhgsi_arm_a/BoardConfig.mk
+++ b/phhgsi_arm_a/BoardConfig.mk
@@ -3,8 +3,6 @@ include device/phh/treble/board-base.mk
 
 ifeq ($(BOARD_SYSTEMIMAGE_PARTITION_RESERVED_SIZE),)
 BOARD_SYSTEMIMAGE_PARTITION_SIZE := 1313583104
-else
-BOARD_SYSTEMIMAGE_PARTITION_RESERVED_SIZE := 8388608
 endif
 
 TARGET_USES_64_BIT_BINDER := false
-- 
2.25.1

